api/api_test.go

```diff
@@ -1038,7 +1061,7 @@ func TestAPI_UnixSocket(t *testing.T) {
 	socket := filepath.Join(tempDir, "test.sock")
 
 	c, s := makeClientWithConfig(t, func(c *Config) {
-		c.Address = "unix://" + socket
+		c.Address = "localhost"
```

Comment:
Is this change necessary?

Reply:
no!  thank you.  that's a remnant. fixed in 8f223080c027847cd4b3363fa32a66780e5cca5c